src/components/proposalActions.jsx

```diff
@@ -511,7 +514,7 @@ const ProposalVote = ({
                   early={
                     earlyExecuteMinionType(proposal) &&
                     proposal.yesShares >= quorumNeeded &&
-                    !proposal?.status === 'NeedsExecution'
+                    proposal?.status === 'NeedsExecution'
```

Comment:
this logic, `!proposal?.status === 'NeedsExecution'` , will return `false` everytime as `boolean never === string` (except for '')

updating the condition so it flows through the prop correctly.